blockmanager.go

```diff
@@ -374,11 +392,6 @@ func (b *blockManager) SyncHeight() int64 {
 // later than the final checkpoint or some other reason such as disabled
 // checkpoints.
 func (b *blockManager) findNextHeaderCheckpoint(height int64) *chaincfg.Checkpoint {
-	// There is no next checkpoint if checkpoints are disabled or there are
-	// none for this current network.
-	if cfg.DisableCheckpoints {
-		return nil
-	}
```

Comment:
Verified this can't cause checkpoints to be inadvertently used because `headersFirstMode` is only set if `DisableCheckpoints` is false.

Reply:
(I'm noting this here because at first glance while reviewing I thought that might be the case, because `findNextHeaderCheckpoint()` is called without a prior direct guard that `DisableCheckpoints == false`, so I had to double check).

Reply:
Right.  I removed it because it can't be hit.